draft-kazuho-httpbis-priority.md

```diff
@@ -387,7 +387,10 @@ Priority information from client requests and server responses to correct or
 amend the precedence to suit it (see {{merging}}).
 
 The Priority header field is an end-to-end signal of the request
-priority from the client or the response priority from the server.
+priority from the client or the response priority from the server.  If
+intermediaries want to specify prioritizaton on a multiplexed HTTP connection,
+it SHOULD use a reprioritizaton frame and SHOULD NOT change the priority header
+field.
```

Comment:
How about promoting this sentence to the last sentence of the paragraph that you changed above?

I think that the sentence would fit better (as we discuss the different use-cases of the two signals) rather than here.

Reply:
Thanks, done.